.github/workflows/deployment.yaml

```diff
@@ -27,16 +27,22 @@ jobs:
     - run: yarn install --frozen-lockfile
     - run: yarn build
 
-    - name: Publish to NPM
+    - name: Publish to NPM registry
       run: yarn publish --access public
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+    - name: Publish to Github registry
+      run: |
+        echo registry=https://npm.pkg.github.com/farmerconnect > .npmrc
+        echo //npm.pkg.github.com/farmerconnect/:_authToken=${{ secrets.GITHUB_TOKEN }} >> .npmrc
+        yarn publish --access public --non-interactive
```

Comment:
Unfortunately, the following does not work:
`yarn config set registry https://npm.pkg.github.com/farmerconnect`

Don't know if ther is an easier way to override the .npmrc created in the `actions/setup-node@v2` step